mitodl/bootcamp-ecommerce
# Using djoser common functionality for password case insenitivity issue

#### What are the relevant tickets?
fixes: #1295 

#### What's this PR do?
This PR accomplishes the following tasks: 
- Adds ol-django-authentication app to BootCamp Ecommerce.
- Removes unused Djoser code

#### How should this be manually tested?
This PR can be tested using the following steps

- Create a BootCamp online account
- Try to change password via Forgot Password and verify it works
- Try to change password via Forgot Password, Change case of email and verify it works
- Try to change password via Account page and verify Valid notification is sent, Password is changed, User remains signed in
